homeassistant/components/flipr/__init__.py

```diff
@@ -0,0 +1,109 @@
+"""The Flipr integration."""
+import asyncio
+from datetime import timedelta
+import logging
+
+from flipr_api import FliprAPIRestClient
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_EMAIL, CONF_PASSWORD
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+)
+
+from .const import CONF_FLIPR_ID, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+SCAN_INTERVAL = timedelta(minutes=60)
+
+
+PLATFORMS = ["sensor"]
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
+    """Set up Flipr from a config entry."""
+    _LOGGER.debug("async_setup_entry starting")
+
+    hass.data.setdefault(DOMAIN, {})
+
+    coordinator = FliprDataUpdateCoordinator(hass, entry)
+
+    await coordinator.async_refresh()
```

Comment:
```suggestion
    await coordinator.async_config_entry_first_refresh()
```

and you should be able to remove the check for `coordinator.last_update_success`

Reply:
OK, I did not know this method.
This will raise me to 100% test coverage :)